public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
+			if (format === 'regex') {
+				return '.*';
```

Comment:
if you want All -> to be `.*` instead of all values in a `(val1|val2|val3) expression then you can specify a custom allValue